.travis.yml

```diff
@@ -53,15 +53,10 @@ before_install:
   - cd ../../
 
 script:
-  # Test the app
-  - cd ../../
-  - sh -c "if [ '$CODECHECK' = '1' ]; then ./occ app:check-code $APP_NAME -c private -c strong-comparison; fi"
-  - sh -c "if [ '$CODECHECK' = '2' ]; then ./occ app:check-code $APP_NAME -c deprecation; fi"
-  - cd apps/$APP_NAME/
-
+  - composer install
   # Run phpunit tests
   - cd tests/Unit
-  - sh -c "if [ '$INTEGRATION' != '1' -a '$JSTESTS' != '1' -a '$CODECHECK' != '1' -a '$CODECHECK' != '2' ]; then phpunit --configuration phpunit.xml; fi"
+  - sh -c "if [ '$INTEGRATION' != '1' -a '$JSTESTS' != '1' -a '$CODECHECK' != '1' -a '$CODECHECK' != '2' ]; then vendor/bin/phpunit --configuration phpunit.xml; fi"
```

Comment:
You can run integration with github actions, no?
https://github.com/nextcloud/app-tutorial/blob/137dc55c3dbd263d9950c6c1a88d6f0c85253a77/.github/workflows/phpunit.yml#L71

Reply:
I'm not going to do that for now, as this app is the only app testing Nextcloud with Oracle, and I dont want to waste time on getting that to run in github actions